addon/actions/action.js

```diff
@@ -111,7 +115,8 @@ export default EmberObject.extend({
   requestUrl() {
     let modelName = this.get('modelName');
     let id = this.get('instance') ? this.get('model.id') : null;
-    let snapshot = this.get('model')._createSnapshot(this.get('config.adapterOptions'));
+    let snapshot = this.get('model')._createSnapshot();
```

Comment:
I see. Their private interface was not a wrapper;
We can use `this.get('model')._internalModel.createSnapshot({ adapterOptions: this.get('config.adapterOptions') });`

Reply:
changed